server/src/pages/shot/text-tool.js

```diff
@@ -228,41 +241,28 @@ exports.TextTool = class TextTool extends React.Component {
 
   onKeyDown(e) {
     this.adjustX(e);
-    if (e.key === "Enter") {
-      e.preventDefault();
-    }
   }
 
-  onPaste(e) {
-    window.setTimeout(() => {
-      // Remove element tags, line breaks and new line seen after pasting text
-      while (this.textInput.current.firstElementChild) {
-        this.textInput.current.removeChild(this.textInput.current.firstElementChild);
-      }
-      this.textInput.current.textContent = this.textInput.current.textContent.replace(/\r?\n|\r/g, "");
-      this.adjustX(e);
-    });
-  }
-
-  onKeyUp(e) {
-    // Fix to remove <br> element inserted on press of space bar inside contenteditable div
-    while (this.textInput.current.firstElementChild) {
-      this.textInput.current.removeChild(this.textInput.current.firstElementChild);
-    }
+  onInput(e) {
```

Comment:
Can we get rid of `onKeyDown` then?

Reply:
need onkeydown to handle arrow keys when no input is there

Reply:
or is it handled by text input handlers? will check

Reply:
yes, looks good. Should be ok to remove